project/application/src/pages/TeamPage/index.js

```diff
@@ -84,26 +89,32 @@ class TeamPage extends Component {
         const headerItemsElements = this.headerItems.map((element, index) => (
             <th key={'team-group-header_' + index}>{element}</th>
         ));
-        const items = programersArr.map((element, index) => (
-            <tr key={'team-member_' + index}>
-                <td>{element.user[0].username}</td>
-                <td>{element.user[0].email}</td>
-                <td>
-                    {checkIsMemberByRole(element.role_collaboration.title) && (
-                        <div className="access_container">{element.role_collaboration.title}</div>
-                    )}
-                    {checkIsAdminByRole(element.role_collaboration.title) && (
-                        <div className="access_container red">{element.role_collaboration.title}</div>
-                    )}
-                </td>
-                <td>
-                    <div>{element.user[0].is_active ? 'Active' : 'Not active'}</div>
-                    {checkIsAdmin() && (
-                        <i onClick={e => this.openEditMiodal(element.user[0])} className="edit_button item_button" />
-                    )}
-                </td>
-            </tr>
-        ));
+        const items = programersArr.map((element, index) => {
+            element.user[0].role = element.role_collaboration.title;
```

Comment:
This assignment is not used anywhere in this scope. Revert back the changes in this block

Reply:
We still need it for the role check.